vir/simd_cvt.h

```diff
@@ -0,0 +1,56 @@
+/* SPDX-License-Identifier: LGPL-3.0-or-later */
+/* Copyright © 2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH
+ *                  Matthias Kretz <m.kretz@gsi.de>
+ */
+
+#ifndef VIR_SIMD_CVT_H_
+#define VIR_SIMD_CVT_H_
+
+#include "simd_concepts.h"
+#if VIR_HAVE_SIMD_CONCEPTS
+#define VIR_HAVE_SIMD_CVT 1
+
+namespace vir
+{
+  namespace detail
+  {
+    template <class From, class To>
+      struct is_simd_convertible
+      : std::false_type
+      {};
+
+    template <class T0, class A0, class T1, class A1>
+      struct is_simd_convertible<stdx::simd<T0, A0>, stdx::simd<T1, A1>>
+      : std::false_type
+      {};
```

Comment:
Where are you using those? `detail` namespace means, that they're an implementation detail, right?

If they're used by some magic, then please put some comment somewhere on the magic being used. Because currently, I would ask to remove them.

Reply:
Right. That was a failed attempt to write something useful and I forgot to remote it.